juzgados.js.org

opensas opened the pull request:
- There is reasonable content on the page (see: [No Content](https://github.com/js-org/dns.js.org/wiki/No-Content))

Check [https://mteyss.github.io/juzgados/](https://mteyss.github.io/juzgados/)

- I have read and accepted the [ToS](http://dns.js.org/terms.html)

Great work. Thanks a lot for your effort!!!

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```juzgados.js.org``` to make it work.
I'll merge after we have "contact".

opensas commented:
Thanks a lot!

I added the CNAME file, I was waiting for your reply.

Once again thanks for your effort

Saludos

Sas

On Wed, May 16, 2018 at 4:57 AM, Stefan Keim <notifications@github.com>
wrote:

> I've already added your requested subdomain to JS.ORGs zonefile. But you
> have to (re-)add a file named CNAME to your branch with the single line
> juzgados.js.org to make it work.
> I'll merge after we have "contact".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/dns.js.org/pull/1997#issuecomment-389429639>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAdZl7kmN9UbZan2wxbOQ0yLKZEFtRiLks5ty9vfgaJpZM4UASXN>
> .
>

indus commented:
Welcoem https://juzgados.js.org
